src/ralph/defaults.py

```diff
@@ -35,9 +38,68 @@ class StorageBackends(Enum):
     FS = "ralph.backends.storage.fs.FSStorage"
 
 
+def get_value(key, default_value):
+    """
+    Get a value based on its key returning the first of (in order):
+        1. Command line argument
+        2. Environment
+        3. Config file
+        4. default_value
+    """
+
+    value = environ.get(key, None)
+    if value is not None:
+        return value
+
+    if CONFIG_JSON is not None and key in CONFIG_JSON:
+        return CONFIG_JSON[key]
+
+    return default_value
+
+
+DEFAULT_LOGGING_CONFIG = {
+    "version": 1,
+    "propagate": True,
+    "formatters": {
+        "ralph_format": {
+            "format": "%(asctime)-23s %(levelname)-8s %(name)-8s %(message)s"
+        },
+    },
+    "handlers": {
+        "ralph_stderr": {
+            "class": "logging.StreamHandler",
+            "level": "INFO",
+            "stream": "ext://sys.stderr",
+            "formatter": "ralph_format",
+        },
+    },
+    "loggers": {
+        "ralph": {
+            "handlers": ["ralph_stderr"],
+            "level": "INFO",
+        },
+    },
+}
+
+
 APP_DIR = Path(environ.get("RALPH_APP_DIR", get_app_dir("ralph")))
-DEFAULT_GELF_PARSER_CHUNCK_SIZE = 5000
-DEFAULT_BACKEND_CHUNCK_SIZE = 500
 ENVVAR_PREFIX = "RALPH"
-HISTORY_FILE = Path(environ.get("RALPH_HISTORY_FILE", APP_DIR / "history.json"))
-FS_STORAGE_DEFAULT_PATH = Path("./archives")
+
+CONFIG_PATH = APP_DIR / ".config"
+CONFIG_JSON = None
+if CONFIG_PATH.is_file():
+    try:
+        with open(CONFIG_PATH) as config_file:
+            CONFIG_JSON = json.load(config_file)
+    except Exception as exc:
+        raise ConfigurationException("Configuration file is not valid") from exc
```

Comment:
You must avoid executing code while loading a module, it may have unexpected side effects. Always wrap code execution in lazy loaded functions.

Reply:
I understand why, however it can't be done without storing the configuration in a global variable or reloading the entire configuration every time, and since the first one does not pass the linting, I chose the latter